CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+# Contributing to `dbt-trino`
+
+## Getting the code
+
+### Installing git
+
+You will need `git` in order to download and modify the `dbt-core` source code. On macOS, the best way to download git is to just install [Xcode](https://developer.apple.com/support/xcode/).
+
+### How to contribute?
+
+If you are not a member of the `starburstdata` GitHub organization, you can contribute to `dbt-trino` by forking the `dbt-trino` repository. For a detailed overview on forking, check out the [GitHub docs on forking](https://help.github.com/en/articles/fork-a-repo). In short, you will need to:
+
+1. Fork the `dbt-trino` repository
+2. Clone your fork locally
+3. Check out a new branch for your proposed changes
+4. Push changes to your fork
+5. Open a pull request against `starburstdata/dbt-trino` from your forked repository
+
+## Setting up an environment
+
+There are some tools that will be helpful to you in developing locally. While this is the list relevant for `dbt-trino` development, many of these tools are used commonly across open-source python projects.
+
+### Tools
+
+These are the tools used in `dbt-trino` development and testing:
+
+- [`tox`](https://tox.readthedocs.io/en/latest/) to manage virtualenvs across python versions. We currently target the latest patch releases for Python 3.7, 3.8, 3.9, and 3.10
+- [`pytest`](https://docs.pytest.org/en/latest/) to define, discover, and run tests
+- [`flake8`](https://flake8.pycqa.org/en/latest/) for code linting
+- [`black`](https://github.com/psf/black) for code formatting
+- [`mypy`](https://mypy.readthedocs.io/en/stable/) for static type checking
+- [`pre-commit`](https://pre-commit.com) to easily run those checks
+- [`changie`](https://changie.dev/) to create changelog entries, without merge conflicts
+- [`make`](https://users.cs.duke.edu/~ola/courses/programming/Makefiles/Makefiles.html) to run multiple setup or test steps in combination. Don't worry too much, nobody _really_ understands how `make` works, and our Makefile aims to be super simple.
```

Comment:
Some tools are not used currently, please open issues if we want to implement them. Black, mypy, pre-commit are very welcome.

Reply:
I will introduce them in separate commits.

Reply:
ok so black doesn't deal with imports?